Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/CheckVersionIntegrityCommand.cs

```diff
@@ -83,14 +93,19 @@ private FileIntegrity CheckFile(AppContentSummaryFile file)
                 return new FileIntegrity(file.Path, FileIntegrityStatus.InvalidVersion);
             }
 
-            string hash = HashCalculator.ComputeFileHash(_localDirectory.Path.PathCombine(file.Path));
+            string hash = HashCalculator.ComputeFileHash(localPath);
```

Comment:
Hash is calculated anyway even if `_checkType != IntegrityLevel.HashChecksum`.

Reply:
fixed

Reply:
Thank you! 😃